helper/dataGeneration-Mongo.js

```diff
@@ -1,41 +1,41 @@
 const fs = require('fs');
-const writeItems = fs.createWriteStream('mongoImages.csv');
+const { images } = require('../cloudinary.js');
+const writeItems = fs.createWriteStream('mongoImages.tsv');
 
 let globalCounter = 99
 let numberOfEntries = 10000100
-let pictureId = 1
+let index = 0
 
-// Items 1-2000 in loremflickr are cat photos, so looping through them with a simple counter
-// will create 2000 unique animal images
 
 console.time('Time to Generate Data: ');
 
-writeItems.write('itemId,pic1Small,pic1Med,pic1Large,pic2Small,pic2Med,pic2Large,\n', 'utf8');
+writeItems.write('itemId\tpic1Small\tpic1Med\tpic1Large\tpic2Small\tpic2Med\tpic2Large\t\n', 'utf8');
 
 
```

Comment:
is this escaping that cloudinary requires?

Reply:
I had to change it from csv to tsv since there is a comma in the cloudinary url, so it was breaking up the url at the wrong point. Double quotes didn't work with mongoimport in that it still broke up the url in the middle where there's a comma, so the only solution was to load and import it as a tab separated value instead of a comma separated value.

Reply:
I had to refactor from a csv to a tsv because there is a comma in cloudinary's image urls and mongoimport kept breaking the url at the comma.